paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationHandler.kt

```diff
@@ -281,7 +281,7 @@ internal class IntentConfirmationHandler(
                     PaymentConfirmationResult.Failed(
                         cause = nextStep.cause,
                         message = nextStep.message,
-                        type = PaymentConfirmationErrorType.Internal,
+                        type = PaymentConfirmationErrorType.Payment,
```

Comment:
This will start reporting everything coming out of `IntentConfirmationInterceptor` from now on. Previously we weren't doing this (even before `IntentConfirmationHandler`) but given there are no longer any UI dependencies, we should report errors coming out of the interceptor flow.